### PR TITLE
HMAN-67 - Populate GET partiesNotified response

### PR DESCRIPTION
**Pull request by markdathornehmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/HMAN-67

### Change description ###
Populate GET partiesNotified response

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/hmc/data/PartiesNotifiedEntity.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
+package uk.gov.hmcts.reform.hmc.data;
```
Comment (RebeccaBaker) [inline src/main/java/uk/gov/hmcts/reform/hmc/data/PartiesNotifiedEntity.java:1]: Not sure we need this class when we have HearingResponseEntity?

File: src/main/resources/db/migration/V20220214_51__HMAN-51-PartiesNotified.sql [hunk 2]
```diff
@@ -0,0 +1,11 @@
+ALTER TABLE ONLY public.hearing_response
+    ADD COLUMN IF NOT EXISTS response_version varchar(60) not null;
+
+ALTER TABLE ONLY public.hearing_response
```
Comment (RebeccaBaker) [inline src/main/resources/db/migration/V20220214_51__HMAN-51-PartiesNotified.sql:4]: hearing_request was no longer required for my ticket, not sure if its still relevant for yours

File: src/main/resources/db/migration/V20220216_66__HMAN-66-Add_Column_PartiesNotifiedDateTime.sql [hunk 3]
```diff
@@ -0,0 +1,2 @@
+ALTER TABLE ONLY public.hearing_response
```
Comment (RebeccaBaker) [inline src/main/resources/db/migration/V20220216_66__HMAN-66-Add_Column_PartiesNotifiedDateTime.sql:1]: if you are merging both HMAN-66 and 67 together - do we need this file when this column is added in [V20220214_51__HMAN-51-PartiesNotified.sql](https://github.com/hmcts/hmc-cft-hearing-service/pull/78/files#diff-b9b01ebe8f881acf5e029fac5f442cd6a6af181c5eafff01c084d10835fe96e9)

**Review by RebeccaBaker — APPROVED** [review]
Approved with minor comments and on condition of a green pr

File: src/main/java/uk/gov/hmcts/reform/hmc/data/PartiesNotifiedEntity.java (hunk 1 above)
Comment (markdathornehmcts) [reply src/main/java/uk/gov/hmcts/reform/hmc/data/PartiesNotifiedEntity.java:1]: Agreed. Will refactor

File: src/main/resources/db/migration/V20220216_66__HMAN-66-Add_Column_PartiesNotifiedDateTime.sql (hunk 3 above)
Comment (markdathornehmcts) [reply src/main/resources/db/migration/V20220216_66__HMAN-66-Add_Column_PartiesNotifiedDateTime.sql:1]: Good point

File: src/main/resources/db/migration/V20220214_51__HMAN-51-PartiesNotified.sql (hunk 2 above)
Comment (markdathornehmcts) [reply src/main/resources/db/migration/V20220214_51__HMAN-51-PartiesNotified.sql:4]: did you mean request_version?

**Comment by RebeccaBaker** [comment]
closing as this was merged in to [HMAN-51 HMAN-52](https://github.com/hmcts/hmc-cft-hearing-service/pull/75)

**Pull request closed without merging**